web/controllers/user_role_controller.ex

```diff
@@ -30,4 +30,10 @@ defmodule CodeCorps.UserRoleController do
         |> render(CodeCorps.ChangesetView, "error.json-api", changeset: changeset)
     end
   end
+
+  def handle_delete(conn, record) do
+    record
+    |> Repo.delete
+    |> @analytics.track(:deleted, conn)
```

Comment:
So we can just pass a `record` along to `ja_resource` and it won't try to do anything else with it, is that right?